server.js

```diff
@@ -0,0 +1,11 @@
+var path = require("path");
+var express = require("express");
+
+var app = express();
+
+app.use(express.static(path.join(__dirname, "dist")));
```

Comment:
When is the web app built and put in the dist directory?

Reply:
The rules for this area in the base webpack configuration that is run with the build script. I'm not sure exactly what is being used to compress and compile it, but I know that heroku runs the build and starts scripts upon initialization.